Source/WebCore/css/parser/CSSPropertyParserWorkerSafe.cpp

```diff
@@ -40,6 +40,12 @@
 #include "CSSParserImpl.h"
 #include "CSSParserTokenRange.h"
 #include "CSSPropertyParser.h"
+#include "CSSPropertyParserConsumer+Angle.h"
+#include "CSSPropertyParserConsumer+Color.h"
+#include "CSSPropertyParserConsumer+Ident.h"
+#include "CSSPropertyParserConsumer+Integer.h"
+#include "CSSPropertyParserConsumer+List.h"
+#include "CSSPropertyParserConsumer+Percent.h"
 #include "CSSPropertyParserHelpers.h"
```

Comment:
Do we still need to import `CSSPropertyParserHelpers` ?

Reply:
That is still needed.